tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/tutorial.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Golden testing in Haskell with hspec-golden
+published: 2019-08-10
+tags: haskell, golden testing, snapshot testing, expect testing
+libraries: hspec-golen-0.1.0.0
+language: haskell
+author: Adrian Vera
+author-name: Adrian Vera
+twitter-profile: adrian_med
+github-profile: medAdrian
+description: In this tutorial we will implement golden tests in Haskell using hspec-golden a super set of the known test framework hspec. A golden test is an Input/Output action that records the result of a test into a file known as the "Golden File".
+---
+
+Golden testing (also known as Golden master testing or Snapshot testing) is pretty similar to unit testing, except that the test output is stored in a separate file known as the "Golden File".
+
+This kind of testing will help us verify modifications made to a reference version of our SUT(Subject Under Test) and validate that it has not modified its
```

Comment:
This changed from:
```
There are many forms of getting the same result with different approaches, as the [Jest](https://jestjs.io/docs/en/snapshot-testing.html) snapshot testing,
comparing two screenshots as [Wraith](https://github.com/BBC-News/wraith) or with the [Approval testing method](https://dzone.com/articles/approval-tests-alternative).
```

@CristhianMotoche wrote: `Looks good. Although, I think we should do some minor changes. Could you please add those paragraphs so that I can left some comments?`

Reply:
@CristhianMotoche Could you please tell me if is this change approved, or if i need to change something.